### PR TITLE
clearer, more specific wording about sponsorship and contributing

### PR DESCRIPTION
**Pull request by indirect**
make it clear that everyone's contributions are welcome, and make it clear that Ruby Together funds contributors, but does not maintain the project directly

### CODE REVIEW

**Comment by indirect** [comment]
@segiddins r?

**Comment by indirect** [comment]
@segiddins any feedback now?

@wycats can you weigh in here as well?

**Comment by segiddins** [comment]
I think this is much better

**Comment by wycats** [comment]
@indirect This is a pretty big improvement.

My understanding is that the Bundler project would be happy to allow any company sponsoring significant amount of development time a similar kind of blurb and call to action. I think it's reasonable for Bundler to have a policy allowing sponsors to describe themselves in their own words, as long as it's a general policy that would apply to any sponsor and not specially crafted for RubyTogether.

@indirect I believe you're cool with that?

**Comment by indirect** [comment]
@wycats that's correct, and there will be a public policy document to that effect as soon as it's drafted and approved by the core team. 👍

File: README.md [hunk 1]
```diff
@@ -49,6 +44,13 @@ To get in touch with the Bundler core team and other Bundler users, please see [
 
 If you'd like to contribute to Bundler, that's awesome, and we <3 you. There's a guide to contributing to Bundler (both code and general help) over in [our documentation section](doc/README.md).
 
+While some Bundler contributors are compensated by Ruby Together, the project maintainers make decisions independent of Ruby Together. As a project, we welcome contributions regardless of the author’s affiliation with Ruby Together.
+
+### Supporting
+
+<a href="https://rubytogether.org/"><img src="https://rubytogether.org/images/rubies.svg" width=200></a><br/>
+<a href="https://rubytogether.org/">Ruby Together</a> pays some Bundler maintainers for their ongoing work. As a grassroots initiative committed to supporting the critical Ruby infrastructure you rely on, Ruby Together is funded entirely by the Ruby community. Contribute today <a href="https://rubytogether.org/developers">as an individual</a> or even better, <a href="https://rubytogether.org/companies">as a company</a>, and ensure that Bundler, RubyGems, and other shared tooling is around for years to come.
```
Comment (colby-swandale) [inline README.md]: `Bundler, RubyGems, and other` -> `Bundler, RubyGems and other`

Comment (colby-swandale) [inline README.md]: `, and ensure that Bundler` -> `to ensure that Bundler...`

**Comment by segiddins** [comment]
@indirect ping on Colby's comments?

**Comment by indirect** [comment]
@segiddins you can pry my oxford comma from my cold dead hands. 😆 That said, less commas is probably better, so I updated the wording a bit.

**Comment by indirect** [comment]
@bundlerbot r+

**Pull request merged**